iosHyperskillApp/Podfile

```
@@ -7,5 +7,6 @@ target 'iosHyperskillApp' do
 
   # Pods for iosHyperskillApp
   pod 'GoogleSignIn', '6.1.0'
+  pod 'SwiftLint'
```

Всегда явно указываем версию библиотеки:
```suggestion
  pod 'SwiftLint', '0.47.0'
```